jsapp/js/components/formLanding.es6

```diff
@@ -170,7 +169,7 @@ var FormLanding = React.createClass({
                   {/*<bem.FormView__label m='questions'></bem.FormView__label>*/}
                   <bem.FormView__label m='clone'>
                       <bem.FormView__link m='clone'
-                          data-version-id={item.version_id}
+                          data-version-id={item.uid}
```

Comment:
Should we rename this prop to `data-form-uid`? Or is the `item` a "version" object?